inference/core/interfaces/stream/inference_pipeline.py

```diff
@@ -419,6 +421,10 @@ def init_with_yolo_world(
         )
 
     @classmethod
+    @experimental(
+        reason="Usage of workflows with `InferencePipeline` is experimental feature now. Please report any issues "
```

Comment:
Why experimental? Seems fairly core

```suggestion
        reason="Usage of workflows with `InferencePipeline` is an experimental feature. Please report any issues "
```

Reply:
👍